spec/features/collection_drafts/publish_collection_draft_spec.rb

```diff
@@ -3,6 +3,30 @@
     login
   end
 
+  context 'when publishing a collection draft with newly added forms, values, features when UMM-C is updated' do
+    before do
+      draft = create(:full_collection_draft, user: User.where(urs_uid: 'testuser').first)
+      visit collection_draft_path(draft)
+    end
+
+    context 'when new enum value is added in UMM 1.15.4' do
```

Comment:
I still don't think it's necessary to have this specific test here, but if you want to keep it, I think it should be moved. 
I am not sure where at the moment, but I'm thinking possibly a different file for 'publishing the latest version' (but that is redundant) or possibly in the migrations tests folder, again for 'publishing the latest version' (which is still redundant).

Reply:
Let me know if you come up with another location, open to any ideas